### PR TITLE
updated with license requirement

### PR DESCRIPTION
**Pull request by DevOpsStyle**
Azure Virtual Desktop Multi-Session Session Host can be enrolled in intune. As Multi-Session Device don't have a primary user at least is needed to have a valid device license assigned to the device in order to complete the enrollment following the compliance policy.

Thank you

### CODE REVIEW

**Comment by PRMerger7** [comment]
@DevOpsStyle : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

File: memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ This feature supports Windows 10 or Windows 11 Enterprise multi-session VMs, whi
   - Configured with [Active Directory group policy](/windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy), set to use Device credentials, and set to automatically enroll devices that are Hybrid Azure AD-joined.
   - [Configuration Manager co-management](/configmgr/comanage/overview).
 - Azure AD-joined and enrolled in Microsoft Intune by enabling [Enroll the VM with Intune](/azure/virtual-desktop/deploy-azure-ad-joined-vm#deploy-azure-ad-joined-vms) in the Azure portal.
+- A valid device only license [Microsoft Intune licensing](/memdocs/intune/fundamentals/licenses.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/azure-virtual-desktop-multi-session.md:73]:
```suggestion
- A valid device-only license. For more information, see [Microsoft Intune licensing](/memdocs/intune/fundamentals/licenses.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DevOpsStyle, kindly commit the suggested change. Many thanks.

**Comment by PRMerger3** [comment]
@DevOpsStyle : Thanks for your contribution! The author(s) have been notified to review your proposed change. @smbhardwaj

**Comment by DevOpsStyle** [comment]
@JohanFreelancer9 committed with your suggestions. Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Smritib17** [comment]
Note that the same issue was raised in the PR: https://github.com/MicrosoftDocs/memdocs/pull/2974 and it has been addressed after getting feedback from the PM. You can close this PR as it is a duplicate.

**Pull request closed without merging**